packages/amplication-client/src/Resource/create-resource-page/CreateResourceForm.tsx

```diff
@@ -380,11 +385,39 @@ const CreateResourceForm = ({ projectId }: Props) => {
                   label="Project"
                   onChange={handleProjectChange}
                 />
+                <HorizontalRule />
+
+                <Panel
+                  panelStyle={EnumPanelStyle.Default}
+                  style={{
+                    marginBottom: "var(--default-spacing)",
+                  }}
+                >
+                  <FlexItem
+                    direction={EnumFlexDirection.Row}
+                    gap={EnumGapSize.Small}
+                    itemsAlign={EnumItemsAlign.Center}
+                  >
+                    <Icon
+                      icon="info_circle"
+                      size="small"
+                      color={EnumTextColor.ThemeGreen}
+                    />
+                    <Text
+                      textStyle={EnumTextStyle.Description}
+                      textColor={EnumTextColor.ThemeGreen}
```

Comment:
Can it be not green? It looks like an approval message rather than a guidance text